pkg/handler/handler.go

```diff
@@ -37,9 +39,18 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// This metric is used to count the number of pods that are mutated for either sts web identity or container credentials
+var webhookPodCount = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "pod_identity_webhook_pod_count",
+		Help: "Indicator to how many pods are using sts web identity or container credentials",
+	}, []string{"Method"},
+)
+
 func init() {
 	_ = corev1.AddToScheme(runtimeScheme)
 	_ = admissionregistrationv1beta1.AddToScheme(runtimeScheme)
+	prometheus.MustRegister(webhookPodCount)
```

Comment:
We already have a place where we register metrics: https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/pkg/handler/middleware.go